src/components/model-authoring/definition-form.tsx

```diff
@@ -114,10 +115,16 @@ export const DefinitionForm = (props: IProps) => {
     } else {
       return (
         <div className={css.buttons}>
-          <button onClick={handleEditSubmit("save and edit previous")}>&lt;&lt; Save &amp; Previous</button>
+          <button className={`${css.saveAnd} ${css.previous}`} onClick={handleEditSubmit("save and edit previous")}>
```

Comment:
Nice - I like the combo of `${css.saveAnd} ${css.previous}` here and `${css.saveAnd} ${css.next}` below.

Reply:
Thanks Doug! :)